experiment/commenter/BUILD

```diff
@@ -72,3 +60,10 @@ go_test(
     importpath = "k8s.io/test-infra/experiment/commenter",
     deps = ["//prow/github:go_default_library"],
 )
+
+# Delete after gazelle stops forcing its creation
```

Comment:
Alternately I (bregrudgingly) added another form of this:
```python
go_image(
   name = "foo",
   binary = "bar",
)
```

For `lang_image` that don't benefit from layering, this is no worse and sidesteps this issue.